### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by mffap**
Fixes various typos across readmes, console, login

### CODE REVIEW

File: console/src/assets/i18n/de.json [hunk 1]
```diff
@@ -232,11 +232,11 @@
       "DELETE_SELF_DESCRIPTION": "Sie sind im Begriff Ihren eigenen Benutzer endgültig zu löschen. Dadurch werden Sie ausgeloggt und Ihr Account wird gelöscht. Diese Aktion kann nicht rückgängig gemacht werden!",
       "DELETE_AUTH_DESCRIPTION": "Sie sind im Begriff Ihren Account endgültig zu löschen. Wollen Sie dies wirklich tun?",
       "TYPEUSERNAME": "Wiederholen Sie '{{value}}', um den Benutzer zu löschen.",
-      "DELETE_BTN": "Entgültig löschen"
+      "DELETE_BTN": "Endgültig löschen"
     },
     "SENDEMAILDIALOG": {
       "TITLE": "Email Benachrichtigung senden",
-      "DESCRIPTION": "Klicken Sie den untenstehenden Button um ein verifizierungs Email an die aktuelle Adresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
+      "DESCRIPTION": "Klicken Sie den untenstehenden Button um ein Verifizierung-Email an die aktuelle Adresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
```
Comment (fforootd) [inline console/src/assets/i18n/de.json:239]:
```suggestion
      "DESCRIPTION": "Klicken Sie den untenstehenden Button um ein Verifizierung-E-Mail an die aktuelle Adresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
```

File: console/src/assets/i18n/de.json [hunk 2]
```diff
@@ -291,12 +291,12 @@
         "DELETE_DESCRIPTION": "Sie sind im Begriff eine Passwortlose Authentifizierungsmethode zu entfernen. Sind sie sicher?",
         "ADD_TITLE": "Passwortlose Authentifizierung",
         "ADD_DESCRIPTION": "Wählen Sie eine der verfügbaren Optionen für das Erstellen einer passwortlosen Authentifizierungsmethode.",
-        "SEND_DESCRIPTION": "Senden Sie sich einen Registrierungslink an Ihre email adresse.",
+        "SEND_DESCRIPTION": "Senden Sie sich einen Registrierungslink an Ihre Email Adresse.",
```
Comment (fforootd) [inline console/src/assets/i18n/de.json:294]:
```suggestion
        "SEND_DESCRIPTION": "Senden Sie sich einen Registrierungslink an Ihre E-Mail Adresse.",
```

File: console/src/assets/i18n/de.json [hunk 3]
```diff
@@ -640,7 +640,7 @@
     },
     "ADDTRIGGER": "Trigger hinzufügen",
     "TIMEOUT": "Timeout",
-    "TIMEOUTINSEC": "Timout in Sekunden",
+    "TIMEOUTINSEC": "Timeout in Sekunden",
```
Comment (fforootd) [inline console/src/assets/i18n/de.json:643]: Timeout is not a proper German word, or am I wrong?

File: console/src/assets/i18n/de.json (hunk 1 above)
Comment (fforootd) [reply console/src/assets/i18n/de.json:239]: See also here https://www.duden.de/suchen/dudenonline/E-Mail

File: console/src/assets/i18n/de.json (hunk 3 above)
Comment (mffap) [reply console/src/assets/i18n/de.json:643]: As technical term fine, imo

File: console/src/assets/i18n/de.json (hunk 1 above)
Comment (fforootd) [reply console/src/assets/i18n/de.json:239]: as well as #3679

**Review by fforootd — APPROVED** [review]

**Pull request merged**